osu.Game/Rulesets/Mods/IReadFromConfig.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Game.Configuration;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public interface IReadFromConfig

```

Comment:
Can we get a little XML doc on this interface? A short one like all the others should be totally enough.